### PR TITLE
Comment NL32

### PR DESCRIPTION
**Pull request by Aurige**
Addition of AccessFacilityList to CommonFacilityGroup (so that we can use it in VehicleType and others entities) Addition of “automaticRamp”, “wheelchairLift”, “slidingStep” to the AccessFacilityEnum

### CODE REVIEW

**Comment by skinkie** [comment]
image

Pull request is already in master.

**Comment by Aurige** [comment]
OK for doc update

File: xsd/netex_framework/netex_reusableComponents/netex_facility_support.xsd [hunk 1]
```diff
@@ -150,10 +150,13 @@ Rail transport, Roads and Road transport
 		<xsd:restriction base="xsd:NMTOKEN">
 			<xsd:enumeration value="unknown"/>
 			<xsd:enumeration value="lift"/>
+			<xsd:enumeration value="wheelchairLift"/>
```
Comment (ue71603) [inline xsd/netex_framework/netex_reusableComponents/netex_facility_support.xsd:153]: The changes in netex_facility_support were already made. Only netex_facility_version is missing. I think we can drop the changes to netex_facility_support.xsd

Comment (ue71603) [inline xsd/netex_framework/netex_reusableComponents/netex_facility_support.xsd:153]: The changes in netex_facility_support were already made. Only netex_facility_version is missing. I think we can drop the changes to netex_facility_support.xsd

Comment (ue71603) [reply xsd/netex_framework/netex_reusableComponents/netex_facility_support.xsd:153]: @skinkie

Comment (ue71603) [reply xsd/netex_framework/netex_reusableComponents/netex_facility_support.xsd:153]: @skinkie

Comment (ue71603) [reply xsd/netex_framework/netex_reusableComponents/netex_facility_support.xsd:153]: sorry it was Christophe @Aurige

Comment (ue71603) [reply xsd/netex_framework/netex_reusableComponents/netex_facility_support.xsd:153]: sorry it was Christophe @Aurige

**Review by skinkie — APPROVED** [review]

**Review by skinkie — CHANGES_REQUESTED** [review]
If you want it in in CommonFacilityGroup it must be removed from SiteFacilityGroup. Please decide!

**Comment by skinkie** [comment]
@Aurige if you don't want to have it changed, just close the pull request.

**Comment by skinkie** [comment]
Can be closed, no changes.

**Comment by trurlurl** [comment]
Removing "needs documentation update" label since closed without having been merged.

**Pull request closed without merging**